components/admission-webhook/main.go

```diff
@@ -484,7 +483,7 @@ func mutatePods(ar v1beta1.AdmissionReview) *v1beta1.AdmissionResponse {
 
 	crdclient := getCrdClient()
 	list := &settingsapi.PodDefaultList{}
-	err := crdclient.List(context.TODO(), &client.ListOptions{Namespace: pod.Namespace}, list)
+	err := crdclient.List(context.TODO(), list, &client.ListOptions{Namespace: pod.Namespace})
```

Comment:
Dumb question why was this reordering needed?

Reply:
the function signature changed. when moving to v1 the original ordering did not work. below is the signature.

https://github.com/kubernetes-sigs/controller-runtime/blob/3cb67228604239d3cd764b41719565bb4a46add6/pkg/client/client.go#L258